Dockerfile

```diff
@@ -49,7 +49,6 @@ RUN set -xe; \
     su lobsters -c "gem update"; \
     su lobsters -c "gem install rake -v 12.3.2"; \
     su lobsters -c "bundle install --no-cache"; \
-    su lobsters -c "bundle add puma --version '~> 3.12.1'"; \
```

Comment:
Lobsters already specifies Puma in its Gemfile, hence I've remove it here :)

Reply:
awesome. I had to go digging into the puma thing, I believe it was broken upstream for a while and thats how it landed in there.. good catch. it's def no longer needed 

https://github.com/utensils/docker-lobsters/commit/9b011778e4a2d808c6ee073c3cffa08e2cecba5f#diff-3254677a7917c6c01f55212f86c57fbfL58